Complications/Wearable/src/main/java/com/example/android/wearable/wear/complications/SmallImageDataSourceService.kt

```diff
@@ -92,7 +93,11 @@ class SmallImageDataSourceService : SuspendingComplicationDataSourceService() {
             .setTapAction(tapAction)
             .build()
 
+    /**
+     * Different SmallImage styles supported for complications.
+     */
     private enum class Case {
-        PHOTO, ICON
+        PHOTO, // Cropped photo-style image
+        ICON // Bounded icon-style image
```

Comment:
Are these definitions from the complication somewhere? 

I would have thought it was more about bitmap Vs shape with tint. But I'm not sure.